.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy to Netlify
+
+on:
+  push:
+    branches: ['main', 'master']
```

Comment:
Can we leave it just only `main`?

Reply:
thank you! Fix all your comments on https://github.com/rosle/react-nimble-survey/pull/9/commits/a093c9046e886cabe7762f5e63c05779a6b56d9f ✨